pkg/core/table/cpin.go

```diff
@@ -0,0 +1,114 @@
+// // Copyright (c) 2021 by library authors. All rights reserved.
+// // Use of this source code is governed by a BSD-style
+// // license that can be found in the LICENSE file.
+
+// // Implements TCG Storage Core Table operations on Locking SP tables
+
+package table
+
+import (
+	"github.com/open-source-firmware/go-tcg-storage/pkg/core"
+	"github.com/open-source-firmware/go-tcg-storage/pkg/core/method"
+	"github.com/open-source-firmware/go-tcg-storage/pkg/core/uid"
+)
+
+type CPINInfoRow struct {
+	// 5.3.2.12.1 This is the unique identifier of this row in the C_PIN table.
```

Comment:
@yogev-lb : TBH, the stream of long-winded chunks of the spec copy-pasted as comments actually reduces code readability. perhaps a reference to section 5.3.2.12 once, in a struct comment, would be sufficient. in any case someone who really needs to do some digging (e.g. cross-checking types and sizes) will need to open the spec, so...

Reply:
updated